Library/Formula/curl.rb

```diff
@@ -47,7 +51,7 @@ def install
     if build.with?("libressl") && build.with?("openssl")
       ohai <<-EOS.undent
       --with-openssl and --with-libressl are both specified and
-      curl can only use one at a time; proceeding with openssl.
+      curl can only use one at a time; proceeding with libressl.
```

Comment:
Yes. ` cURL ` is rather special in that it explicitly and happily supports Secure Transport, so we default to that to avoid dropping extra mandatory deps on anyone. Not dead-set against changing the default when *both* are declared as is the situation here, but I fear we may see some problems for end-users in doing so.